graphdatascience/query_runner/session_query_runner.py

```diff
@@ -37,7 +38,8 @@ def __init__(
         self._gds_arrow_client = arrow_client
         self._resolved_protocol_version = ProtocolVersionResolver(db_query_runner).resolve()
         self._progress_logger = QueryProgressLogger(
-            self._gds_query_runner.run_cypher, self._gds_query_runner.server_version
+            lambda query, database: self._gds_query_runner.run_cypher(query=query, database=database),
```

Comment:
did you have to make this `lambda` or was it a choice?

Reply:
It could have been a method reference like the one you did for the `neo4j_query_runner` as well

Reply:
Yes, but it contained the bug that it relied on positional arguments rather than named arguments, so we would pass the database value into the `params` argument on `run_cypher`